content/source/docs/cloud/workspaces/variables.html.md

```diff
@@ -95,42 +86,52 @@ There may be cases when a workspace contains conflicting variables of the same t
+
+### 3. Workspace-Specific Variables
 
 Workspace-specific variables always overwrite variables from variable sets that have the same key. Refer to [overwrite variables from variable sets](/docs/cloud/workspaces/managing-variables.html#overwrite-variable-sets) for details.
 
-### 2. Non-Global Variable Sets
+### 4. Non-Global Variable Sets
 
 Non-global variables are only applied to a subset of workspaces in an organization.
 
 When non-global variable sets have conflicting variables, Terraform Cloud uses values from the variable set that was applied most recently. For example, if you apply Variable Set A and then apply Variable Set B to the same workspace a week later, Terraform Cloud will use any conflicting variables from Variable Set B. This will be the case regardless of which variable set has been edited most recently; Terraform Cloud only considers the date that each variable set is applied to the workspace when determining precedence.
 
-### 3. Global Variable Sets
+### 5. Global Variable Sets
 
 Non-global variable sets always take precedence over global variable sets that are applied to all workspaces within an organization. This is true regardless of when the global variable set was applied.
 
-### 4. Variable Files
-
-When variables from [committed `.auto.tfvars` files](#loading-variables-from-files) have the same key as existing variables in the Terraform Cloud workspace, the variables applied to the workspace overwrite variables from the files.
-
+### 6. `auto.tfvars` Variable Files
 
+Variables in the Terraform Cloud workspace and variables provided through the command line always overwrite variables from files ending in `auto.tfvars` with the same key.
 
 ### Precedence Example
 
 Consider an example workspace that has the following variables applied:
 
-| Name | Scope | Date Applied | ACCESS_KEY | ACCESS_ID | VAR1 | KEY1 | VAR2|
-|-----|-------|-------|-----|-------|-------|--------|--------|------|
-| Variables | Workspace-Specific | 10/1 | `g47fh474` | `874hf7u4` | `h` |     |     |
-| Variable Set A | Non-Global | 10/4 |   |   | `y` | `x` |   |   |
-| Variable Set B | Global | 10/20 |   |     |   |  `z` | `a` |
+| Name           | Scope              | Date Applied | ACCESS_KEY | ACCESS_ID  | VAR1 | KEY1 | VAR2 | foo |
+|----------------|--------------------|--------------|------------|------------|------|------|------|-----|
+| TF_VAR_foo=8   | Run-Specific       |              |            |            |      |      |      | `8` |
+| -var="foo=9"   | Run-Specific       |              |            |            |      |      |      | `9` |
+| Variables      | Workspace-Specific | 10/1         | `g47fh474` | `874hf7u4` | `h`  |      |      | `1` |
+| Variable Set A | Non-Global         | 10/4         |            |            | `y`  | `x`  |      | `2` |
+| Variable Set B | Global             | 10/20        |            |            |      | `z`  | `a`  | `3` |
```

Comment:
Okay so, I was thinking about this example and about how to make it possibly clearer. One thing is that I want to change name to Source because I think maybe it'll be clearer where these things are coming from (and I tried to use language that would match the precedence headings above) see what you think about that. And THEN I was thinking that maybe it'd be nice to use a real variable name instead of just "foo". We have a Learn tutorial where we are setting up a kubernetes deployment and one of the properties we have to set is the number of replicas: https://learn.hashicorp.com/tutorials/terraform/cdktf-applications

Anyway, I also updated the example a bit with that in mind. Let me know what you think of that, too and/or if we should use a different example variable :-) 
```suggestion
| Source            | Scope              | Date Applied | ACCESS_KEY | ACCESS_ID  | VAR1 | KEY1 | VAR2 | REPLICAS |
| ----------------- | ------------------ | ------------ | ---------- | ---------- | ---- | ---- | ---- | -------- |
| Command Line      | Run-Specific       | 10/23        |            |            |      |      |      | `4`      |
| Local Environment | Run-Specific       | 10/23        |            |            |      |      |      | `3`      |
| Workspace UI      | Workspace-Specific | 10/1         | `g47fh474` | `874hf7u4` | `h`  |      |      | `1`      |
| Variable Set A    | Non-Global         | 10/4         |            |            | `y`  | `x`  |      | `2`      |
| Variable Set B    | Global             | 10/20        |            |            |      | `z`  | `a`  | `3`      |
```

Reply:
Sounds good!